scripts/linters/pylint_extensions.py

```diff
@@ -1917,6 +1917,77 @@ def process_tokens(self, tokens):
                         'single-line-pragma', line=line_num)
 
 
+class SingleSpaceAfterIfElifWhileChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks that there is a single space
+    between an `if`, `elif`, or `while` statement and a bracket `(` if
+    a bracket is used.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'single-space-after-if-elif-while'
+    priority = -1
+    msgs = {
+        # ERIC: Figure out how to name this message. Does 'C0029' make sense?
+        'C0029': (
```

Comment:
C0029 is the way to add ID:
C = `Convention` 0029 = `Id for this check`

Can you please update the "Next id to use to 30" on line 18?

More info here: http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html#writing-an-ast-checker